client/components/FormattingBar/formattingBar.scss

```diff
@@ -161,7 +161,7 @@
         z-index: 2;
     }
     &.full-screen-width:not(.floating) {
-        width: 100vw;
+        width: 100%;
```

Comment:
This is a big visual improvement, but the popover should probably be the full 1200px width of `header-parts` rather than the width of only the `formatting-bar-component` — the difference between the two is determined by the text of the connection status indicator, which can jump between `connected`, `saving`, and `disconnected`. We don't want the popover to reflow when one of those transitions occurs.

I can pop this open and figure out what the best way to implement it will be — I suspect by nesting the connected indicator inside of the formatting bar itself.

Reply:
Sure thing! I figured someone who has worked with the formatting bar before is probably better placed to take care of this dynamic resizing anomaly. We also had a chat recently about the 'Connected' part going away and probably living someplace else on the pub view. But that's later in the roadmap I suppose.